goonstation.dme

```diff
@@ -25,6 +25,7 @@ var/datum/preMapLoad/preMapLoad = new
 #include "code\__secret_public.dme"
 
 // BEGIN_INCLUDE
+#include "_std\macros\appearance.dm"
```

Comment:
no.

Reply:
No?

Reply:
This is not how _std folder includes are done

Reply:
I probably should've been paying attention when Adhara was cleaning all this stuff up. Moved to the right file.